apps/extension/src/pages/settings/subscription/pricing-card.tsx

```diff
@@ -0,0 +1,92 @@
+// import logoIcon from "data-base64:~assets/icon512.png";
+import { Membership } from "@agreeto/api/types";
+import { AgreeToLogo, Button } from "@agreeto/ui";
+import type { FC } from "react";
+import { HiCheckCircle } from "react-icons/hi";
+
+import { trpcApi } from "~features/trpc/api/hooks";
+
+type Props = {
+  plan: Extract<Membership, "PRO" | "PREMIUM">;
+  period: "monthly" | "annual";
+  onBuyClick?: () => void;
+};
+
+interface Feature {
+  text: string;
+}
+const features: Feature[] = [
+  {
+    text: "Select & share availability",
+  },
+  {
+    text: "Create & confirm Holds",
+  },
+  {
+    text: "2 Time Zones",
+  },
+  {
+    text: "3 Calendars",
+  },
+  {
+    text: "Add colleagues",
+  },
+  {
+    text: "Multi-language support",
+  },
+];
+
+export const PricingCard: FC<Props> = ({ plan, period }) => {
+  const title = plan === Membership.PRO ? "Pro Plan" : "Premium Plan";
+  const price = period === "monthly" ? 6 : 48;
+
+  const { mutate: createCheckoutSession } =
+    trpcApi.stripe.checkout.create.useMutation({
+      onSuccess({ checkoutUrl }) {
+        chrome.tabs.create({ url: checkoutUrl });
+      },
+    });
```

Comment:
This might need to redirect to an intermediate route on web depending on if the content script is happy or not